tokenizer_ts/src/bytePairEncode.ts

```diff
@@ -1,17 +1,70 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-/**
- * Convert a Uint8Array to a string
- * @param uint8Array
- * @returns string
- */
-export function uint8ArrayToString(uint8Array: Uint8Array): string {
-  return Array.from(uint8Array)
-    .map(num => num.toString())
-    .join("_");
+const enum Constant {
+  // we have 48 bits per level, we can safely bitwise encode 32 bits at a time,
+  // so this works in two passes
+  BytesPerLevel = 6,
+}
+
+const binaryMapKey = (k: Uint8Array): number => {
+  const lower = k[0] | (k[1] << 8) | (k[2] << 16);
+  const upper = 0xFFFFFF * (k[3] | (k[4] << 8) | (k[5] << 16));
```

Comment:
as mentioned offline, should be multiplied by `0x1000000`

Reply:
Why?

Reply:
If the input is `0x00 FF FF FF` (I don't think this exists currently since that's not a valid text prefix, but in theory) then it would have the same key as `0x01 00 00 00`.

Reply:
I can add this to my PR, is that ok @connor4312 ?

Reply:
Yep!